lib/productions/container.js

```diff
@@ -16,6 +16,19 @@ function inheritance(tokeniser) {
   return { colon, inheritance };
 }
 
+/**
+ * Parser callback.
+ * @callback parserCallback
+ * @param {import("../tokeniser.js").Tokeniser} tokeniser
+ * @param {...*} args
+ */
+
+/**
+ * A parser callback and optional option object.
+ * @typedef AllowedMember
+ * @type {[parserCallback, object?]}
+ */
+
```

Comment:
```suggestion
/**
 * Parser callback.
 * @callback ParserCallback
 * @param {import("../tokeniser.js").Tokeniser} tokeniser
 * @param {...*} args
 */

/**
 * A parser callback and optional option object.
 * @typedef AllowedMember
 * @type {[ParserCallback, object?]}
 */

```

PascalCase for types.